app/classes/Langchecker/Project.php

```diff
@@ -28,36 +28,26 @@ public static function getReferenceLocale($website)
     }
 
     /**
-     * Return supported files for website
+     * Return supported locales for a file in website. If not defined,
```

Comment:
I though you were referring explicitly to the first param, using website is fine